v3/integrations/nrsecurityagent/nrsecurityagent.go

```diff
@@ -63,7 +69,7 @@ func InitSecurityAgent(app *newrelic.Application, opts ...ConfigOption) error {
 
 	appConfig, isValid := app.Config()
 	if !isValid {
-		return fmt.Errorf("Newrelic application value cannot be read; did you call newrelic.NewApplication?")
+		return fmt.Errorf("Newrelic  application value cannot be read; did you call newrelic.NewApplication?")
```

Comment:
Is that a typo or do you intend the extra space to be added to the error string?

Reply:
yes extra space is typo.

Reply:
ok we can fix it on release.